src/posts/blog/3-hepa-filters/hepa-filters.tsx

```diff
@@ -0,0 +1,145 @@
+        the aerosol density in the room by 90%, which they predict would reduce
+        the transmission of COVID by <strong>4-6x</strong>. It’s not as good as
+        being outside, but if you must have people over inside, an air purifier
+        make the interaction safer.
+      </p>
+
+      <p>
+        We’ve added air purifiers as an option in the “Environment” question on
+        the calculator.
+      </p>
+
+      <p>
+        If you decide to buy an air purifier for COVID purposes, here’s some
+        things to keep in mind:
+      </p>
+
+      <ul>
+        <li>
+          Make sure the purifier has a HEPA filter - these are rated to remove
+          99.97% of airborne particles.
+        </li>
+        <li>
+          Watch out for lesser filters - products labeled as "HEPA-type,"
+          "HEPA-like," "HEPA-style" or "99% HEPA" do not satisfy the HEPA
+          standard!
+        </li>
+        <li>
+          Central heat / AC systems don't work for this. These typically use
+          MERV-rated filters. These are significantly less effective at removing
+          small particles than HEPA-rated filters (the best MERV rating,
+          MERV-16, merely removes 75% of particles.
+        </li>
+        <li>
+          Air flow matters! In the paper, they used a flow rate equivalent to
+          cleaning the whole room worth of air 5 times an hour. Purifiers will
+          typically rate their flow as CADR (Clean Air Delivery Rate) in CFM
+          (cubic feet per minute) or m<sup>3</sup>/h (cubic meters per hour).
+          For a room with 8ft tall ceilings, this works out to a{' '}
+          <strong>CFM rating of 2/3 your room’s square footage</strong> (length
+          * width in feet).
+        </li>
+        <li>
+          For example, my apartment’s living room is{' '}
+          <code>
+            22ft x 12ft = 264 ft<sup>3</sup>
+          </code>{' '}
+          . So I need <code>2/3 * 264 = 176CFM</code>.{' '}
+          <Link to="https://www.amazon.com/LEVOIT-Filtration-Eliminators-Core-P350/dp/B08131HFSG/">
+            This purifier
+          </Link>{' '}
+          advertises a flow rate of 141 CFM, so I would need more than one for
+          my space.
+        </li>
+        <li>
+          Of course, this is microCOVID, so we've made a little calculator to do
+          this math for you:
+        </li>
+      </ul>
+      <HepaCalculator />
+      <p />
+
+      <p>And remember, all the wisdom from the summer still applies:</p>
+
+      <ul>
+        <li>
+          Limit how many people you interact with indoors. Avoid gatherings with
+          more than a few people.
+        </li>
+        <li>
+          Wear a mask when you are out of the house and especially when you are
+          indoors with strangers.
+        </li>
+        <li>
+          Talk to your friends about what their COVID practices are - encourage
```

Comment:
Does this mean "talk to the friends you are seeing"?

Reply:
Ostensibly? But tbh, I'm happy if you talk to your other friends too.

Reply:
Yeah I worry the broader statement doesn't make sense in the context of the post and the calculator, which is primarily about you reducing your COVID risk. And there is a lot of talk about "pandemic shaming." I think "talk to your friends" is definitely something one can do in a good way, but the way it's written here, it could be interpreted as "I need to go out and tell all my friends they are doing it wrong." If we wanna make a broader statement, I think we should be more precise about what we do/don't mean by it.

I'll leave the final call to you.

Reply:
Updated.